css/index.css

```diff
@@ -49,7 +50,61 @@
     font-weight: bolder;
     letter-spacing: 3mm;
 }
-
+body {
+    background-color: rgb(240, 221, 161);
+}
+#about {
+    text-align: center;
+    background-color: rgba(194, 137, 32, 0.918);
+    border: 3px dotted black;
+    margin: 10%
+}
+#experience {
+    text-align: center;
+    border: 3px dotted black;
+    margin: 5%;
+    background-color: rgb(194, 165, 69);
+}
+.headings {
+    text-align: center;
+    font-family: fantasy;
+    text-decoration: underline;
+}
+#skillsList {
+    border: 5px solid rgb(83, 58, 10);
+    margin-left: 40%;
+    margin-right: 40%;
+}
+#connect {
+    text-align: center;
+}
+.leavingMessages{
+    color:rgb(37, 26, 5)
+}
+.name {
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    color: rgb(233, 94, 2);
+}
+.email{
+    color: rgb(233, 94, 2);  
+}
+.messages{
+    color: rgb(233, 94, 2);
+}
+.submit{
+    background-color: rgb(170, 112, 36);
```

Comment:
did you know that the RGB color [170, 112, 36](https://convertingcolors.com/rgb-color-170_112_36.html) is a dark color, and the websafe version is hex [996600](https://convertingcolors.com/hex-color-996600.html), so a complement of this color would be [36, 94, 170](https://convertingcolors.com/rgb-color-36_94_170.html), and the grayscale version is [121, 121, 121](https://convertingcolors.com/rgb-color-121_121_121.html)? :)

Reply:
I didn't. Thank you!